cmd/vsphere-xcopy-volume-populator/api/v1beta1/types.go

```diff
@@ -0,0 +1,63 @@
+package v1alpha1
+
+import (
+	meta "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/scheme"
+)
+
+var SchemeGroupVersion = schema.GroupVersion{
+	Group: "forklift.konveyor.io",
+	// Version: "v1alpha1",
+	// workaround mtv code
```

Comment:
Please not mention the MTV but use Forklift.
What workaround?

Reply:
removing this whole api dir is not needed anymore